### PR TITLE
add tcp listen to be able to use only zmq_ex

### PR DESCRIPTION
**Pull request by Dreat**
Added handling when nothing is listening at specified port, so you can
communicate using only elixir code now!

### CODE REVIEW

File: lib/zmq_ex.ex [hunk 1]
```diff
@@ -7,13 +7,22 @@ defmodule ZmqEx do
...
     spawn(fn -> rec_loop(socket, printer_process) end)
 
-    if send? do
-      send_loop(socket)
+    # if send? do
+    send_loop(socket)
+    # end
+  end
+
+  defp connect(port, opts) do
```
Comment (fabiankrol) [inline lib/zmq_ex.ex]: Let's make explicite function for server(listen+accept) and client(connect) and propagate it to the top. For example:
`def start_srv do .... listen()` and `def start_cli do ... connect()`.

Comment (fabiankrol) [reply lib/zmq_ex.ex]: I'll be easier to create tests when we can have predefined roles upfront

Comment (nenros) [reply lib/zmq_ex.ex]: i thought it would be easier if it would be something like that https://github.com/half-t/zmq_ex/pull/3

Comment (fabiankrol) [reply lib/zmq_ex.ex]: @Dreat @nenros 
I think this short-term plan can apply
1. Fix&merge this pr
2. Introduce CI
3. Add simple tests: start listener, connect, pass some msgs 
4. Introduce gen servers

2&3 can be replaced 
3 can(/~~should~~?) be a part of 1 ;)

**Pull request closed without merging**